### DISCUSSION TITLE
Sort settings in Search results web part

### DISCUSSION THREAD

**Discussion by shirleysham** [microsoft-search/pnp-modern-search#2937] [Q&A]
Hi,
Can you please advise how the Sort settings works in PnP Search?
ie. where is the dropdown list of field names derived from? If it's a custom value, should it be a slot name or a slot field? Either way, the sort settings doesn't seem to work for me.

![image](https://user-images.githubusercontent.com/121912343/233251668-1b12b63a-2dd7-4bd2-a2d0-3d5470819a40.png)

Thanks!

**Comment by kasperbolarsen** [comment]
It has to be a field, and the field must be in Selected Properties AND it must be sortable ( many fields are not, and must be mapped to a RefinableString in order to become sortable)
